addons/core/XEH_postInit.sqf

```diff
@@ -2,75 +2,34 @@
-		["2BNB Flecktarn - Pilot",[[],[],["BWA3_MP7","","","",["BWA3_40Rnd_46x30_MP7",40],[],""],["BWA3_Uniform_Crew_Fleck",[["BWA3_Beret_HFlieger",1],["ACE_elasticBandage",5],["ACE_packingBandage",10],["ACE_adenosine",1],["ACE_CableTie",2],["ACE_epinephrine",2],["ACE_IR_Strobe_Item",1],["ACE_morphine",2],["ACE_tourniquet",4],["ACE_MapTools",1],["ACE_Flashlight_XL50",1],["ACE_EarPlugs",1],["BWA3_40Rnd_46x30_MP7",2,40],["SmokeShellOrange",1,1],["SmokeShellGreen",1,1]]],[],[],"BWA3_Knighthelm","",[],["ItemMap","ItemGPS","TFAR_anprc152","ItemCompass","ACE_Altimeter",""]]] call ace_arsenal_fnc_addDefaultLoadout;
-		["2BNB Flecktarn - Rifleman",[["BWA3_G36A3","","","BWA3_optic_CompM2",["BWA3_30Rnd_556x45_G36",30],[],""],[],["BWA3_P8","","","",["BWA3_15Rnd_9x19_P8",15],[],""],["BWA3_Uniform2_sleeves_Fleck",[["BWA3_Beret_PzGren",1],["ACE_elasticBandage",5],["ACE_packingBandage",10],["ACE_adenosine",1],["ACE_CableTie",2],["ACE_EarPlugs",1],["ACE_epinephrine",2],["ACE_IR_Strobe_Item",1],["ACE_morphine",2],["ACE_tourniquet",4],["ACE_MapTools",1],["ACE_Flashlight_XL50",1]]],["BWA3_Vest_Rifleman_Fleck",[["BWA3_30Rnd_556x45_G36",12,30],["SmokeShell",4,1],["HandGrenade",2,1],["SmokeShellGreen",1,1],["SmokeShellBlue",1,1]]],["BWA3_TacticalPack_Fleck",[["ACE_EntrenchingTool",1]]],"BWA3_M92_Fleck","",[],["ItemMap","ItemGPS","TFAR_anprc152","ItemCompass","ACE_Altimeter",""]]] call ace_arsenal_fnc_addDefaultLoadout;
-		//-------------------------------------------------------------------------------
+	BNB_LoadoutUnits = [];
+	publicVariable "BNB_LoadoutUnits";
+	if (isServer) then
+	{
+		{
+			if (_x getVariable "IsLoadout" == true) then 
+			{
+				BNB_LoadoutUnits pushBack _x;
+				publicVariable "BNB_LoadoutUnits";
+				_x setPos [0,0,0];
+				hideObjectGlobal _x;
+				_x disableAI "all";
+				_x enableSimulationGlobal false;
+			};
+		} forEach allUnits; //Difficult to filter without causing the TFAR error - shouldn't matter too much for now as only happens once at start and Zeuses usually spawn AI instead of having it preplaced in mission, sort out eventually.
+		BNB_LoadoutUnits joinSilent createGroup sideLogic;
+		{
+		_x removeCuratorEditableObjects [BNB_LoadoutUnits, false];
+		} forEach allCurators;
 	};
+	{
+			_role = _x getVariable "LoadoutRole";
+			[_role, getUnitLoadout _x] remoteExec ["ace_arsenal_fnc_addDefaultLoadout", 0, true];
+	} forEach BNB_LoadoutUnits; 
```

Comment:
This would also mean you're not wasting resources having units at 0,0,0. Even if simulation is disabled, it still consumed some stuff especially if there are many units